astropy/config/tests/test_configs.py

```diff
@@ -17,56 +17,59 @@
 
 
 def test_paths():
-    assert 'astropy' in paths.get_config_dir()
```

Comment:
Can it break anyone's code that there no longer is a default?

Reply:
I shouldn't have changed any defaults, I will double check why I have changed this test.